svgdigitizer/svgplot.py

```diff
@@ -336,10 +572,35 @@ def axis_label(axis):
             return labels[-1]
 
         return {
-            self.xlabel: axis_label(self.xlabel),
-            self.ylabel: axis_label(self.ylabel),
+            axis_symbol: axis_label(axis_symbol) for axis_symbol in self.axis_symbols
         }
 
+    @property
+    @cache
+    def _grouped_ref_points(self):
+        r"""
+        Return the reference points grouped by axis symbol.
+        """
+
+        # group ref_points by axis
+        keyfunc = lambda x: str(x.label).split(":")[0][
+            :-1
```

Comment:
Why do we drop the last character?

Reply:
This is removing the number which is not needed for grouping.